### ISSUE TITLE
1.10.1 and 1.11.x Crashes handling items without details

### ISSUE THREAD

**Issue by KnightCaptainU7** [exult/exult#591]
Please see this discussion for details:
https://github.com/exult/exult/discussions/590

It seems that my work-in-progress mod has uncovered a regression in handling of shapes that do not have names (or certain details?). This had been working fine previously, where Exult handled items without names without crashing.

**Comment by KnightCaptainU7** [comment]
I tried adding details to the shape of Dupre's armour, including the name, defensive value of the item, and paperdoll tab details, but no luck.

**Comment by KnightCaptainU7** [comment]
When I removed Dupre from the party and killed him, the shape of his armour did not appear on the corpse.

**Comment by KnightCaptainU7** [comment]
Using the latest snapshot from today (2024-12-03) the problem persists.
[stderr.txt](https://github.com/user-attachments/files/18001183/stderr.txt)
[stdout.txt](https://github.com/user-attachments/files/18001184/stdout.txt)

**Comment by marzojr** [comment]
Can you share your patch dir so we can debug?

**Comment by KnightCaptainU7** [comment]
TL;DR user error.

I might have caused this issue in my mod. About a year ago I re-numbered all my artwork to have specific "groups". This changed the shape numbers of both armours I added, but I had not updated the initgame.dat nor the paperdol_info.txt.

11xx Humanoids
12xx Monsters / Non-Humanoids
13xx Everything else.
